modules/test/company-sample-data-generation-test/src/testIntegration/java/com/liferay/company/sample/data/generation/test/CompanySampleDataGenerationTest.java

```diff
@@ -243,17 +243,28 @@ private void _addUsers(
 		}
 	}
 
+	private void _exportCompanyData(String hostName) throws Exception {
+		Path companyCSVFilePath = _outputDirPath.resolve(_COMPANY_CSV);
+
+		try (BufferedWriter companyBufferedWriter = new BufferedWriter(
```

Comment:
@slnn Although it's OK to use `FileWriter` here, actually the `Files` API supports:

```
Files.newBufferedWriter(companyCSVFilePath, StandardOpenOption.CREATE, StandardOpenOption.APPED);
```

or 

```
Files.newBufferedWriter(companyCSVFilePath, _openOptions);

...

private static final OpenOption[] _openOptions = {StandardOpenOption.CREATE, StandardOpenOption.APPEND);
```

Reply:
OK, I see, do you want me to resend the pull to apply the Files API?

Reply:
@slnn I'm also SFing other parts, so I will modify it together.